templates/default/less/variables.less

```diff
@@ -232,4 +232,23 @@
 @bs-datetimepicker-secondary-border-color: #ccc;
 @bs-datetimepicker-secondary-border-color-rgba: rgba(0, 0, 0, 0.2);
 @bs-datetimepicker-primary-border-color: white;
-@bs-datetimepicker-text-shadow: 0 -1px 0 rgba(0, 0, 0, 0.25);
\ No newline at end of file
+@bs-datetimepicker-text-shadow: 0 -1px 0 rgba(0, 0, 0, 0.25);
+
+//== Glyph
+@il-glyph-color: @link-color;
+@il-glyph-hover-color: @link-hover-color;
+@il-glyph-highlighted-color: @brand-warning;
+@il-glyph-highlighted-hover-color:  darken(@brand-warning, 15%);
+
+//== Counter
+@il-counter-padding: 2px 4px;
+@il-counter-font-size: 9px;
+@il-counter-margin-left: -4px;
+
+//== Status Counter
+@il-counter-status-bg: lighten(@mid-gray, 40%);
+@il-counter-status-margin-top: 10px;
+
+//== Novelty Counter
+@il-counter-novelty-bg: @brand-warning;
+@il-counter-novelty-margin-top: -5px;
```

Comment:
At the beginning of the project one idea was to use bootstrap classes and variables as much as possible. Now we get these very specific variables like "il-counter-novelty-margin-top" which will probably never be reused. Does it make sense to define lots of variables that will only be used in one place?